### PR TITLE
Fix using double quotes as attribute wrapper

### PR DESCRIPTION
**Pull request by rowanseymour**
Currently `a(href="don't")` becomes `<a href='don\'t'>` which isn't valid and should be `<a href='don&#39;t'>`.

If using double quotes for attributes then `a(onclick='alert("foo")')` currently becomes `<a onclick="alert(\"foo\")">` which again isn't valid and should be `<a onclick="alert(&quot;foo&quot;)">`

### CODE REVIEW

File: hamlpy/parser/elements.py [hunk 1]
```diff
@@ -190,10 +191,10 @@ def attr_wrap(val):
 
         return " ".join(rendered)
 
-    @staticmethod
-    def _escape_attribute_quotes(v, attr_wrapper):
+    @classmethod
```
Comment (norkans7) [inline hamlpy/parser/elements.py:194]: we need to update https://github.com/nyaruka/django-hamlpy/pull/96/files#diff-971f769c231cfc0bbf4b9a88f9eff1ec4b736a8ed724a1402f1ad7c26c6f8181R183 ?

Comment (norkans7) [reply hamlpy/parser/elements.py:194]: And probably update tests that really use `render_attributes`?

Comment (rowanseymour) [reply hamlpy/parser/elements.py:194]: I'm not following... why do we need to update that line?

Comment (norkans7) [reply hamlpy/parser/elements.py:194]: I was asking to use the class instead of the instance self?

Comment (norkans7) [reply hamlpy/parser/elements.py:194]: ok

**Review by norkans7 — APPROVED** [review]

**Pull request merged**